### ISSUE TITLE
Can We Change Pan Speed

### ISSUE THREAD

**Issue by kassemEzz** [yanwsh/videojs-panorama#72]
Hi Wensheng Yan. Thank you for the great plugin.
I wanted to ask if its possible to increase\decrease the speed at which the video pans to the left, right, up Thanks

**Comment by yanwsh** [comment]
Do you mean desktop or mobile?

**Comment by kassemEzz** [comment]
for desktop when moving the mouse around.

**Comment by yanwsh** [comment]
No, on desktop, the moving speed depends on user's mouse moving speed. So if user moves faster, then the moving speed will be faster, and vice verse.

**Comment by yanwsh** [comment]
But it's possible to close it.

```javascript
        player.panorama({
             clickAndDrag: true
        });
```

**Comment by kassemEzz** [comment]
Thanks for your explanation.
I thought there was an option in three.js called "panSpeed"? cant we use that?
If you allow me, I can send you link to a website that have a 360 video and moves slowly when you move the mouse.

**Comment by yanwsh** [comment]
Yes, sure, could you send me the link, then I can take a look at it.

**Comment by kassemEzz** [comment]
The wesbite is https://with.in/ and the 360 image is the big slider right at the top of the page.

**Comment by yanwsh** [comment]
The website looks very cool, so do you want it look like that? When mouse move, the video view will move slowly? 

That's a very good suggestion.

**Comment by kassemEzz** [comment]
It is a very nice website. I think they using A-frame, Three.js and a custom script (trailer.js) which does the magic

**Comment by yanwsh** [comment]
This is supported on v1.0.0, please check [here](https://github.com/yanwsh/videojs-panorama/releases/tag/v1.0.0-beta)
